src/manifolds/FixedRankMatrices.jl

```diff
@@ -496,6 +635,49 @@ the size of matrices on this manifold ``(m,n)``.
 """
 @generated representation_size(::FixedRankMatrices{m,n}) where {m,n} = (m, n)
 
+@doc raw"""
+    retract(M, p, X, ::OrthographicRetraction)
+Compute the OrthographicRetraction on the [`FixedRankMatrices`](@ref) `M` by finding
+the nearest point to ``p + X`` in
+
+```math
+    p + X + N_{p}\mathcal M \cap \mathcal M
+```
+
+where ``N_{p}\mathcal M `` is the Normal Space of ``T_{p}\mathcal M ``.
+
+If ``X`` is sufficiently small, then the nearest such point is unique and can be expressed by
+
+```math
+    q = (U(S + M) + U_{p})(S + M)^{-1}((S + M)V^{\mathrm{T}} + V^{\mathrm{T}}_{p}),
```

Comment:
Is Eq. (4.9) in Absil and Malik supposed to give the same thing? It looks more computationally efficient in this case.

Reply:
Yes it gives the same thing, but in Absil and Malick they consider the SVD with orthogonal matrices U and V (i.e U,V are square matrices). But in Manifolds.jl we consider U and V as Stiefel matrices, so our SVD and corresponding tangent space look different than in Absil and Malick. Therefore, we cannot use this form of the orthographic retraction directly.

Reply:
I see, thank you for this explanation.